lms/static/sass/course/_student-notes.scss

```diff
@@ -23,6 +23,14 @@ $divider-visual-primary: ($baseline/5) solid $gray-l4;
 $divider-visual-secondary: ($baseline/10) solid $gray-l4;
 $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
 
+%notes-tab-control {
```

Comment:
NOTE: basic clean up of the UI @clrux and I previously touched for viewing all notes.

* Notes-specific abstraction
* visual padding/alignment
* input styling syncing across note UI